pkgs/by-name/no/noita-entangled-worlds/package.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, pkgs }:
+
+pkgs.stdenv.mkDerivation rec {
```

Comment:
1. This should be `stdenvNoCC` if you aren't compiling anything.
2. Currently, the preferred way to refer to `version` from another attribute is to use the `(finalAttrs: { ... })` pattern instead of `rec`. See #293452 for examples.